gem/spec/spec_helper.rb

```diff
@@ -13,7 +13,7 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 
-require_relative '../lib/earlgrey'
+require 'earlgrey/cli'
```

Comment:
I'm not convinced either. `require_relative` guarantees we're testing what we intend to test, not whatever happens to be on the load path.

the only coupling here is that the gem name is expected to remain constant. there's no other practical loss.